### DISCUSSION TITLE
Publicando pacotes com dependências privadas

### DISCUSSION THREAD

**Discussion by luizgoebel** [balta-io/forum#234] [Tira Dúvidas] [answered]
Estou recebendo o erro abaixo:
![image](https://user-images.githubusercontent.com/72111314/213883323-d03a9f68-77c8-4d2c-9374-7c05fd1b22e5.png)
![image](https://user-images.githubusercontent.com/72111314/213883351-0c7d3e81-dad3-4c0a-9653-af6cf9c43319.png)
![image](https://user-images.githubusercontent.com/72111314/213883364-8eb39283-53c2-4255-b323-ef75f91b1694.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @luizgoebel, este erro é referente a permissão, seu `TOKEN_GITHUB` no **Secrets** do repositório está correto?

**Reply by luizgoebel** [reply]
Bom dia André; está correto sim,

![image](https://user-images.githubusercontent.com/72111314/214279080-df302dfb-b31d-4ce4-b479-9df8323a86b2.png)
![image](https://user-images.githubusercontent.com/72111314/214279124-9ac3dd2a-efe0-4314-a570-8fc027201c47.png)
![image](https://user-images.githubusercontent.com/72111314/214279204-50de40f4-ddc6-4d13-a4e3-53e289991b2b.png)
![image](https://user-images.githubusercontent.com/72111314/214279311-877d0806-3d63-4a1b-a40c-dff468697b77.png)

**Reply by andrebaltieri** [reply]
Bom dia, @luizgoebel, tenta gerar outro então, deve ter dado algum problema na hora de copiar e colar... o erro diz que não teve permissão para baixar os pacotes do GitHub Packages privado.
